changes/4626.fixed

```diff
@@ -0,0 +1 @@
+Fixed VRFs Prefix invalid filter link query param name.
```

Comment:
Let's make sure release-note fragments are useful/comprehensible to the end user. Perhaps?

```suggestion
Fixed incorrect "Prefixes" link from VRF detail view.
```